cmd/gateway/azure/gateway-azure.go

```diff
@@ -1318,3 +1318,9 @@ func (a *azureObjects) DeleteBucketPolicy(ctx context.Context, bucket string) er
 func (a *azureObjects) IsCompressionSupported() bool {
 	return false
 }
+
+// GetMetrics - no op
+func (a *azureObjects) GetMetrics(ctx context.Context) (*minio.GatewayMetrics, error) {
+	logger.LogIf(ctx, minio.NotImplemented{})
+	return &minio.GatewayMetrics{}, minio.NotImplemented{}
+}
```

Comment:
this function can be moved to gateway-unsupported.go and overridden only in the case of s3 gateway